projects/frontend/data-pipelines/gui/projects/data-pipelines/src/lib/vdk-data-pipelines.module.ts

```diff
@@ -17,17 +17,12 @@ import { LottieModule } from 'ngx-lottie';
 
 import { DpDatePickerModule } from 'ng2-date-picker';
 
-import {
-    ClarityModule,
-    ClrDatagridModule,
-    ClrSpinnerModule,
-} from '@clr/angular';
-
-import { VdkComponentsModule } from '@versatiledatakit/shared';
+import { ClarityModule } from '@clr/angular';
 
 import {
-    TaurusSharedFeaturesModule,
-    TaurusSharedNgRxModule,
+    VdkSharedComponentsModule,
+    VdkSharedFeaturesModule,
+    VdkSharedNgRxModule,
```

Comment:
I renamed them for consistency and because there were conflicts in SC UI, because the end host application imports both `@versatiledatakit/shared` and `@taurus/shared` which provided same classes and there were two options, to introduce aliases in imports like
```
import { TaurusSharedFeaturesModule as VdkSharedFeaturesModule } from '@versatiledatakit/shared';
import { TaurusSharedFeaturesModule } from '@taurus/shared';

@NgModule({
    imports: [
        ...,
        VdkSharedFeaturesModule.forRoot(),
        TaurusSharedFeaturesModule.forRoot(),
        ....
    ]
})
```
or rename modules that comes from '@versatiledatakit/shared'.
Personally I prefer the second option because we are in the early phase of open-sourcing and would be better to make clear boundaries and interfaces to be easier in future and serve both internal and external consumers in the best way.